### ISSUE TITLE
Summarizing VegaStrike

### ISSUE THREAD

**Issue by BenjamenMeyer** [vegastrike/Vega-Strike-Engine-Source#156]
Let's create a few pieces of documentation to describe Vega Strike for newer contributors:

- [ ] Describe Vega Strike in 100 words or less (multiple sentences).
- [ ] Describe Vega Strike in a single sentence.

Asking some of our older contributors/members to help with this as they have the historical knowledge of the project.

The outcome of this should show up (a) on the GitHub Wiki Page, and (b) on the official VS Website and/or wiki.

If we already have these, great - let's make them more visible.

**Comment by BenjamenMeyer** [comment]
Assigning on backlog as it's not really tied to a release.

**Comment by stephengtuggy** [comment]
How about:

Vega Strike is a single-player 3D space simulator game, with an expansive universe, that allows you to trade; fight; explore; and more, as you choose -- all from the cockpit of your very own ship.

**Comment by evertvorster** [comment]
From my playing of the game, the features that really stand out are the realism in the game engine, the size of the universe and the beautiful art. 

My single sentence summary of Vega Strike would be: 
A spaceflight simulator with accurate physics in an immense and beautiful universe, where you are in control of your destiny.

Multiple sentences would be something like this:
Experience life on the coldest sea of all. The vastness of solar systems is like a grain of sand compared to the Universe that is yours to explore in Vega Strike. Game play is open-ended and aimed at accuracy in simulation. Meet interesting life forms in exquisitely beautiful surroundings. Trade, work or pirate your way to a vast array of pilot-able space ships.

**Comment by pyramid3d** [comment]
A suggestion in 118 words:

Vega Strike is a GPL 3D OpenGL Action RPG space sim that allows a player to trade and bounty hunt. You start in an old beat up Wayfarer cargo ship, with endless possibilities in front of you and just enough cash to scrape together a life. Yet danger lurks in the space beyond.

Vega Strike Upon The Coldest Sea features a fully dynamic universe that advances independent of the player’s actions. Become a trader, pilot, or defender of humanity against alien invasion! With dozens of ships and upgrades to choose from, a great variety of systems, planets and stations to visit and a pinch of tragicomedy, the choice is yours. Be sure, your game will be unique!

Source (my blog): https://openteq.wordpress.com/2020/02/22/vega-strike-upon-the-coldest-sea-0-5-3-code-release/

**Comment by stephengtuggy** [comment]
@pyramid3d Yes, I remember that being the classic game summary. Good enough. :-)

Just one potential update: Don't players start in a Llama now, rather than a Wayfarer?

**Comment by Loki1950** [comment]
Re Wayfarer that's just a class of ship not a specific ship name the Llama is just a small cargo ship that protect it's self moderately'

**Comment by stephengtuggy** [comment]
@Loki1950 Ah, gotcha.

**Comment by nabaco** [comment]
Please tell me once we reach a decision. I'll update the description in the website

**Comment by pyramid3d** [comment]
I think in old times, the class name was "Wayfarer" and later became the "Llama", but not so sure about that. I thought so because our wiki usually lists vessel classes, not names: https://wiki.vega-strike.org/Vessel:Llama

Though it may also be as  @Loki1950  says (Wayfarer class, Llama ship name).
The canon document only mentions the Llama as "Venerable LIHW built armed light cargo shuttle".

**Comment by stephengtuggy** [comment]
Why don't we just change it to Llama, so that it's less confusing?

**Comment by ministerofinformation** [comment]
Wayfarer is deprecated. The Wayfarer name predates the regularization of LIHW naming schemes.
The specific name of the ship you start in is the "Schlimazel" - but that's probably not what should go in the blurb unless you're looking to select for a particular sense of humor as a prerequisite :) . It's not clear to me that an outsider would even really appreciate ship class names, no matter what they are, but would understand "beat up tramp freighter" or its idiomatic equivalent without specific mentioning of the type of craft in question. This is in contrast to an externally established IP universe, where the type of ship in question may actually be part of the appeal (e.g. the LucasArts space sim franchise).

**Comment by stephengtuggy** [comment]
So it sounds like we have something like this:

Vega Strike is a GPL 3D OpenGL Action RPG space sim that allows a player to trade and bounty hunt. You start in a beat-up tramp freighter, with endless possibilities in front of you and just enough cash to scrape together a life. Yet danger lurks in the space beyond.

Vega Strike Upon The Coldest Sea features a fully dynamic universe that advances independent of the player’s actions. Become a trader, pilot, or defender of humanity against alien invasion! With dozens of ships and upgrades to choose from, a great variety of systems, planets and stations to visit and a pinch of tragicomedy, the choice is yours. Be sure, your game will be unique!

**Comment by BenjamenMeyer** [comment]
I like the second paragraph a lot, though I' lean more towards the first paragraph as something like:

> Vega Strike is a GPL 3D OpenGL Action RPG Space Simulation Game engine that provides the ability to build vast universes to explore, battle, and defend. Build games like Vega Strike Upon The Coldest Sea, Privateer WCU, and more.

The following vist VS:UtCS very well:

> Vega Strike Upon The Coldest Sea features a fully dynamic universe that advances independent of the player’s actions. Become a trader, pilot, or defender of humanity against alien invasion! With dozens of ships and upgrades to choose from, a great variety of systems, planets and stations to visit and a pinch of tragicomedy, the choice is yours. Be sure, your game will be unique!

**Comment by stephengtuggy** [comment]
@BenjamenMeyer I see what you mean. Vega Strike is technically the game engine, not the game itself.

**Comment by ministerofinformation** [comment]
Agreed.  It's worth employing the necessary nuance to disambiguate the high overlap in nomenclature between -

1. Vega Strike (An OSS game engine intended [and used] for multiple asset sets)

2. [The] Vega Strike Universe, AKA "VSU" (A fictional setting intended for, but not necessarily limited to, one or more game-oriented experiences)

3. Vega Strike: Upon the Coldest Sea (By idiosyncratic naming convention [see also, starting a trilogy with "Episode IV: A New Hope"], the "2nd" primary episode of the four primary episodes implied by the stanza line --> episode structure (each in their own distinct time period, which then may conceivably have other associated secondary episodes developed, e.g. something set during the Rlaan-Aera war 8 years prior to the UtCS start time would still fall under the broader heading of UtCS, whereas the Fraternal War would be under "Icarus descended")

    Icarus descended /
    Upon the coldest sea /
    Foundering inheritance /
    Rimward of Eden
)
and, to date, the only VSU asset set that meaningfully exists :) .

I apologize for the nested parentheticals - I'd blame LISP, but it's clearly just my own fault ;-)

**Comment by pyramid3d** [comment]
It's always a pleasure to read the elaborations of @ministerofinformation. It's like a mini-universe of its own and part of the attraction of the VSU. But my ranting probably shouldn't go into the summary ,-).

**Comment by BenjamenMeyer** [comment]
@ministerofinformation thanks for the information; didn't realize we also had additional stories lines to work with...so yeah, it'll be cool to see those come to fruition and figure out how that'll work out. We might need to do a `VSU` repo for common stuff between those stories...but that's something for another time once we get other things figured out and can start focusing on story development for our assets.

**Comment by ministerofinformation** [comment]
I think, for any foreseeable future, that as long as we do not claim that any of 1, 2, and 3 are the same as another, we are well enough served with sufficient room for flexible future particulars in branding.

**Comment by stephengtuggy** [comment]
@ministerofinformation In that case, we will probably want to take PR #221 a step further, and rename the [entire CMake project](https://github.com/stephengtuggy/Vega-Strike-Engine-Source/blob/affbfcefe3ba2bccaae1bf9acc53190b892cbe04/engine/CMakeLists.txt#L16), as well. From `vsUTCS` to `Vega-Strike`.

Some other parts of the CMake infrastructure will be affected by that as well. It will be a breaking change, so maybe we'd better not do it on 0.6.x.

**Comment by kheckwrecker** [comment]
Closing as web site has been updated to reflect this discussion.
